### DISCUSSION TITLE
Global config for all keyboards

### DISCUSSION THREAD

**Discussion by nullbyto** [sezanzeb/input-remapper#451] [Q&A] [answered]
Hello,

I use `input-remapper` on several computers. And I have setup my `dotfiles` to have all my configs, and I've come into an issue that every keyboard in `input-remapper` specifically has its own preset file in a dedicated folder so for every computer, I have to reconfigure `input-remapper` to that specific keyboard.

What I'm looking for is a _global_ (generic) config file for all keyboards, or auto detect keyboard, which will fix the problem. How can I do that?

**Comment by jonasBoss** [comment, marked as answer]
this is currently not possible. But you can copy the .json files from one device folder into another it should also be possible to create a link, that way you don't need to worry about keeping the files in sync.

**Reply by nullbyto** [reply]
Can you please explain further how i would link the json file to multiple device folders?

**Reply by tfulmer1** [reply]
@jonasBoss probably means a symlink:

ln -s existing.conf new.conf

Where existing.conf is the one you want to create a copy of and new.conf is the one you want to be a copy. This is the equivalent of making a "shortcut" in windows. So your new.conf just points to the existing.conf. I you change existing.conf, you don't need to copy that changes to new.conf since that is just pointing back to the existing.conf

**Reply by nullbyto** [reply]
Thank you for the detailed answer tho. I thought there is something special i didn't know about.
